keps/262-ConfigurableFailurePolicy/README.md

```diff
@@ -213,18 +313,23 @@ Consider including folks who also work outside the SIG or subproject.
 type FailurePolicyAction string
 
 const (
-  // Fail JobSet immediately, regardless of maxRestarts.
+  // Fail the JobSet immediately, regardless of maxRestarts.
   FailJobSet FailurePolicyAction = "FailJobSet"
 
+  // Restart the JobSet if the "restart-attempts" annotation is less than
```

Comment:
The annotation is an implementation detail. Just say "Restart the JobSet if the number of restarts didn't reach MaxRestarts" or something to that effect.